vip-real-time-collaboration.php

```diff
@@ -60,3 +64,48 @@
 	// Fire action to indicate that the plugin has loaded.
 	do_action( 'vip_real_time_collaboration_loaded' );
 }, 10, 0 );
+
+/**
+ * Verify that we can initialize the VIP Real-Time Collaboration plugin.
+ *
+ * @global string $wp_version The WordPress version string.
+ *
+ * @return bool true if the plugin can load, false otherwise.
+ */
+function vip_real_time_collaboration_pre_init(): bool {
+	$php_version = phpversion();
+	if ( is_string( $php_version ) && version_compare( $php_version, '8.2', '<' ) ) {
+		add_action( 'admin_notices', function (): void {
+			wp_admin_notice(
+				__(
+					'The VIP Real-Time Collaboration plugin requires PHP 8.2+. The VIP Real-Time Collaboration plugin has been disabled.',
+					'vip_real_time_collaboration'
+				),
+				[ 'type' => 'error' ]
+			);
+		}, 10, 0 );
+		return false;
+	}
+
+	global $wp_version;
+
+	// Account for plugins overriding the $wp_version global, look at Gutenberg.php for reference.
```

Comment:
Very minor, but I'm guessing it capitalized on its own since it did it to me too when I typed it.
```suggestion
	// Account for plugins overriding the $wp_version global, look at gutenberg.php for reference.
```

Reply:
Good catch, it was auto-corrected there.